apps/google-analytics-4/frontend/src/components/sidebar/ChartFooter.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Paragraph, TextLink } from '@contentful/f36-components';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import tokens from '@contentful/f36-tokens';
+import { css } from 'emotion';
+
+interface Props {
+  slugName: string;
+  viewUrl: string;
+}
+
+const styles = {
+  slug: css({
+    color: tokens.gray600,
+    fontSize: tokens.fontSizeS,
+    marginBottom: tokens.spacingM,
+  }),
+};
```

Comment:
my custom style radar is going off – can you include screenshots of this with/without? it's possible there is a f36 way of achieving the desired result without custom one-off like this?

Reply:
Yes, here are screenshots with the styling and without, as well as a screenshot of the legacy GA app so you can see that I was trying to mimic that styling on the slug name. I can achieve the same look by removing the custom styling and using: `<Text fontColor="gray600" fontSize="fontSizeS" marginBottom="spacingM">{slugName}</Text>`. So I can update it to that if you thing I should try to keep the same look as the legacy app. Let me know what you think!

![Screenshot 2023-02-24 at 10 59 12 AM](https://user-images.githubusercontent.com/62958907/221269804-9a349787-61df-4828-9b12-858a51f30755.png)
![Screenshot 2023-02-24 at 10 59 39 AM](https://user-images.githubusercontent.com/62958907/221269809-8b304832-31c5-4ef3-a462-9b2233382439.png)
<img width="344" alt="Screenshot 2023-02-24 at 11 59 52 AM" src="https://user-images.githubusercontent.com/62958907/221269811-898335ae-c76b-4641-b217-19af9381c057.png">